### ISSUE TITLE
Offline support

### ISSUE THREAD

**Issue by bjornte** [terreng/simple-web-server#144]
Offline support would be great, e.g. when I am at the cabin.

In my experience, if the computer is offline (e.g. no Wi-Fi connection), the web server does not serve most resources. I get the “Index of /” page, but other resources hangs (spinning browser icon).

If this won't be supported, tips for other offline solutions would be greatly appreciated. I will try [this one](https://superuser.com/a/1626770/287154) for now (MacOS).

**Comment by terreng** [comment]
Hmm. I just tried turning off the WiFi on my mac but I wasn't able to recreate this issue.. Everything still worked. 

1. You're using a localhost address right? (localhost or 127.0.0.1)
2. Are you using the latest version of Simple Web Server? (1.2.8)
3. Are the resources you're trying to load html pages? Maybe they're trying to load additional resources from the internet and that's why it isn't working. (You could test with just a basic text file / image as the file and see if that works)

**Comment by terreng** [comment]
Just checking in about this... I'm not able to recreate this issue, so there's nothing I can do at this time. I'm closing this issue for now, but I'm happy to reopen it if I hear back from you.

**Comment by bjornte** [comment]
Hi Terren,
Apologies for the belated reply on my part. Thank you for having a look at
my issue. After more testing on my part, I suspect the problem might be
that the files are under the Mac's “CloudStorage/Dropbox” folder. In that
folder, MacOS seems to be removing local files rather aggressively, and I
wasn't aware of this since Apple forced Dropbox to move the files long
after I started using Dropbox. So in reality, the files are not stored
locally when I try to access them.

Again sorry that I made you have a look at this problem when in fact it had
nothing to do with your app.
Best, Bjørn T
